rosdep/python.yaml

```diff
@@ -2859,6 +2859,17 @@ python-pyaudio:
   fedora: [pyaudio]
   gentoo: [dev-python/pyaudio]
   ubuntu: [python-pyaudio]
+python-pycurl:
+  debian:
```

Comment:
Since the package is available on all currently supported Debian packages the debian rule can be expressed more simply as

`debian: [python-pycurl]`